docs/examples/custom-json/README.md

```diff
@@ -183,22 +188,26 @@ inference_request = types.InferenceRequest(
             shape=[len(inputs_bytes)],
             datatype="BYTES",
             data=[inputs_bytes],
-            parameters=types.Parameters(content_type="str")
+            parameters=types.Parameters(content_type="str"),
         )
     ]
 )
 
 inference_request_g = converters.ModelInferRequestConverter.from_types(
-    inference_request,
-    model_name=model_name,
-    model_version=None
+    inference_request, model_name=model_name, model_version=None
 )
 
 grpc_channel = grpc.insecure_channel("localhost:8081")
 grpc_stub = dataplane.GRPCInferenceServiceStub(grpc_channel)
 
 response = grpc_stub.ModelInfer(inference_request_g)
-response
+
+print(f"full response:\n")
+print(response)
+# retrive text output as dictionary
+output = json.loads(response.outputs[0].contents.bytes_contents[0])
```

Comment:
Similar to the above, you could run this as:

```python
import json

from mlserver.grpc.converters import ModelInferResponseConverter
from mlserver.codecs.string import StringRequestCodec

# Note that here we just convert from the gRPC types to the MLServer types
inference_response = ModelInferResponseConverter.to_types(response)
raw_json = StringRequestCodec.decode_response(inference_response)
output = json.loads(raw_json)
```

Reply:
This isn't working as the following line:
```
inference_response = ModelInferResponseConverter.to_types(response)
```
is calling the following, which the `inference_response` output for the subsequent line of the scripts will be `None`
https://github.com/SeldonIO/MLServer/blob/fac90c83a3536dbbea9e7075c590e33ace424537/mlserver/grpc/converters.py#L301

Reply:
Damnit, you're completely right. There's no implementation for that method yet. 

One to add for the next version of MLServer. While that's not there though, what you already have for the gRPC block is probably your best bet :+1: